blog/2023-09-15-teams-feature-overview/index.mdx

```diff
@@ -0,0 +1,101 @@
+Inviting users to collaborate on the same private team grants them the ability to share without the exporting and copy/paste gymnastics. 
+
+### A source of what, now?
+Simply put, a "source of truth" is a single place where all the information about a project is stored. 
+In our case, we wanted to make sure that teams have a single place to view and collaborate on machine visualizations, and that place is a team-based project in the Studio.
+This prevents confusion and allows for a shared context between all members of the team. With a proper source of truth configured, we can avoid situations like this: 
+
+![true misery](/blog/2023-09-15-file-confusion.png)
+
+
+
+### But what about source control? Shouldn't that be the source of truth?
+Ideally, yes, the team's repository is the single source of truth for all application logic, but not every member of the team is a developer, nor should they all have access to source code.
+That said, we know teams need to ensure their machines in the Studio are always in lockstep with the rest of their source, so we added the ability to [import machines directly from GitHub](https://stately.ai/docs/import-from-github).
+We're also working on a full-fledged GitHub Sync feature, allowing users to save machines as commits back to their source repos! Check out Anders' demo showing off the feature [here](https://www.youtube.com/watch?v=WelBycMg2eA&t=1293s).
+
+
+
+### Superpower Pro features
+Having teams allows other features to shine. For example, we all know the value of collaboration, but more users naturally add more potential for mistakes to happen, even with proper roles assigned. To protect against mistakes, we've added the ability to [lock machines](https://stately.ai/docs/lock-machines), preventing teammates from accidentally overwriting or deleting machines.
+
+![locked machine](./lockedScreenshot.png)
+
+
+Team members often need to manage and track variants of the same machine, so we added the ability to create [machine versions](https://stately.ai/docs/versions). Members can edit and save specific checkpoints of their machines and can even see the last person to and save a specific version. Team roles come into play here again by allowing only owners and admins to actually delete versions.
+
+![](./versionScreenshot.png)
+
+
+## FAQ's 
+
+### Who can join my team?
+Anyone with an email address can be invited to a team!
+If they are not already members, they will have to [sign up for a Stately account](https://stately.ai/registry/login?).
+
+### How many users can be on a team? 
+A team can have as many members as there are seats in your Pro or Enterprise plan, except for viewers. 
+Viewers are free to add. 
+
+### Does inviting a user take up a seat on the team?
+The seat is only taken once a user accepts the invite and claims the seat. 
+If the invite was not accepted, the seat remains free. 
+
+
+### Can I swap member's roles and reclaim seats?
```

Comment:
```suggestion
### Can I swap member roles and reclaim seats?
```
or `members' roles` but the non-possessive is simpler and more accurate. If there's only 1 other member whose role you can change then there's no need for plural possession. But there are always multiple [member] roles.

Reply:
an excellent catch, @kevinmaes. i think i was going for plural possessive but fat-fingered the apostrophe in the wrong spot. Agree that it's simpler to just use the singular "member" here anyways.